src/pages/_app.js

```diff
@@ -108,6 +108,7 @@ function MyApp({ Component, pageProps, intercom, admin }) {
         intercom,
         intercom.appId,
         intercom.enabled,
+        irods,
```

Comment:
Looks like GitHub actions is throwing a warning that `tools` is also missing as a dependency here.

Reply:
Ah, good catch, thanks 👍